sormas-backend/src/main/resources/sql/sormas_schema.sql

```diff
@@ -9949,4 +9949,11 @@ ALTER TABLE eventparticipant_history ADD COLUMN archived BOOLEAN;
 
 INSERT INTO schema_version (version_number, comment) VALUES (440, 'Refactor CoreAdo to include archiving #7246');
 
+-- 2022-02-09 Align username handling of Keycloak and legacy login #7907
+DROP INDEX idx_users_username;
```

Comment:
Should we worry about systems which are not using Keycloak and might already have users with non unique usernames, like `NatUser` and `natuser` ?

Reply:
I suggest than that we provide a script in a comment to the issue which can help systems identify if they have this kind of issue before upgrading to the newest version

Reply:
Good idea will do.